File: Python Code/Daily_Flash_Code_0/Week13/16Apr/75-DailyFlash_Solutions/16_Apr_Solution_Three/Python/prog2.py

st = input("Enter String : ")

str1=""
for i in st:
    if(i >='A' and i <= 'Z'):

        ch = ord(i)+32;
        str1+=chr(ch)
    elif(i>='a'and i<='z'):
        ch = ord(i)-32;
        str1+=chr(ch)

print(str1)

File: Python Code/Daily_Flash_Code_0/Week13/16Apr/75-DailyFlash_Solutions/16_Apr_Solution_Three/Java/prog2.java
import java.io.*;

class Demo{
	public static void main (String [] args) throws IOException {
		BufferedReader  br = new BufferedReader(new InputStreamReader(System.in));
		System.out.print("Enter String : ");
		String str = br.readLine();
		String str1 = "";
		for(int i = 0; i < str.length(); i++) {
			char ch = str.charAt(i);
			if(ch >= 'A' && ch <= 'Z') {
				ch+=32;
				str1+=Character.toString(ch);
			} else if(ch >= 'a' && ch <= 'z') {
				ch-=32;
				str1+=Character.toString(ch);
			}
		}
		System.out.println(str1);
	}
}
